chainer/initializers/__init__.py

```diff
@@ -37,7 +37,7 @@ def generate_array(initializer, shape, xp, dtype=None, device=None):
     Args:
         initializer: A callable object that takes :ref:`ndarray` and edits its
             value.
-        shape (tuple): Shape of a return array.
+        shape (int or tuple of int): Shape of a return array.
```

Comment:
I do not think this is in the scope of this PR, but what do you think about this change because "a return array" is not so good to me?
```suggestion
        shape (int or tuple of int): Shape of an initialized array.
```

Feel free to ignore this if inappropriate :)

Reply:
Thanks for the good suggestion!